src/v2/Apps/Artist/Routes/AuctionResults/Components/AuctionFilters/index.tsx

```diff
@@ -1,14 +1,15 @@
 import React from "react"
 import { AuctionHouseFilter } from "./AuctionHouseFilter"
 import { MediumFilter } from "./MediumFilter"
-import { SizeFilter } from "./SizeFilter"
+import { SizeFilter } from "v2/Apps/Components/SizeFilter"
 import { YearCreated } from "./YearCreated"
+import { useAuctionResultsFilterContext } from "../../AuctionResultsFilterContext"
```

Comment:
but this can prob use an absolute path too, anything two relative jumps or more away :)